File: cursos/src/app/services/alumno.service.ts
import { HttpClient, HttpHeaders, HttpParams } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { Observable, map } from 'rxjs';
import { Alumno } from '../models/alumno';
import { CommonService } from './common.service';

@Injectable({
  providedIn: 'root'
})
export class AlumnoService extends CommonService<Alumno> {

  protected override baseEndpoint='http://localhost:8090/api/alumnos/api/';


}

File: microservicios-usuarios/src/main/java/com/developer/pe/service/AlumnoService.java
package com.developer.pe.service;

import java.util.List;
import java.util.Optional;

import com.commons.alumnos.models.entity.Alumno;
import com.developer.pe.services.CommonService;

public interface AlumnoService extends CommonService<Alumno> {
	
	
	public List<Alumno> findByNombreOrApellido(String term);
}
